src/components/pages/DetailedProblemView/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC, useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import { useQuery } from "@apollo/client";
+
+import { GetProblemsQuery, PROBLEMS } from "../ProblemIndex/interface";
+import { ProblemSummaryProps } from "../../molecules/SearchResultTile/interface";
+
+const DetailedProblemView: FC = () => {
+  const params = useParams();
+  const id: any = params.problemId;
```

Comment:
`number`?
```suggestion
  const id: number = params.problemId;
```

Reply:
At first I tried with `number`, but the error occurred. That's why i set `any`. Now when I set as `number` there is the same message: `Type 'string | undefined' is not assignable to type 'number'.
  Type 'undefined' is not assignable to type 'number'`

Reply:
I see, yes params would be strings always, so here I'd go with
```suggestion
  const id: string | undefined = params.problemId;
```

Reply:
I've changed it to: `const id: string = params.problemId || "";`